cmd/estuary-shuttle/main.go

```diff
@@ -91,154 +158,156 @@ func main() {
 	logging.SetLogLevel("rcmgr", "debug")
 
 	app := cli.NewApp()
+	usr, _ := user.Current()
+
+	cfg := config.NewShuttle()
+
 	app.Flags = []cli.Flag{
```

Comment:
Can we abstract all of these flags into the shuttle config? so we can reduce the notice on `main.go` and also, it makes testing easy as we can test these components individually.

```
app.Flags = cfg.GetAppFlags() //  since cfg = config.NewShuttle(), it will return only shuttle app flags. We can do the same for Estuary

Reply:
I think there is an argument for keeping cli stuff out of the config package. It could become a dependency monster if every way to mutate configuration lives in the config package. For testing purposes isn't it enough just to expose the overrideSetOptions function in main?